test/llvm/lifetime-intrinsics/README.md

```diff
@@ -0,0 +1,3 @@
+### Notes
+
+Some of these tests check for temporary variables such as `coerce_tmp_chpl` and `i_lhs_chpl`. We can potentially get rid of those in the future (if needed).
```

Comment:
Can we pick just one of these tests to check temporaries (preferably a pretty simple one) & remove CHECK lines for the temporaries elsewhere please? That would seem to remove the need for this file (and it could be a comment in a particular test instead). I am concerned about the maintenance burden of these tests if many of them depend on temporary names and their existence.